doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -64,98 +64,185 @@ Checklist:
 
-To remove another node from the cluster, call the following command from one of the cluster members:
+    The successful output from the above command may look like the following:
 
-```bash
-mongooseimctl remove_from_cluster RemoteNodeName
-```
+    ```text
+    The node mongooseim2@localhost has successfully left the cluster
+    ```
 
-where `RemoteNodeName` is a name of the node that we'd like to remove from our cluster.
-This command could be useful when the node is dead and not responding and we'd like to remove it remotely.
-The successful output from the above command may look like the following:
+### Removing a node from the cluster
 
-```text
-The node mongooseim2@localhost has been removed from the cluster
-```
+=== "CETS"
 
-### Cluster status
+    A stopped node would be automatically removed from the node discovery table in RDBMS database after some time.
+    It is needed so other nodes would not try to connect to the stopped node.
 
-You can use the following commands on any of the running nodes to examine the cluster
-or to see if a newly added node is properly clustered:
+=== "Mnesia"
 
-```bash
-mongooseimctl mnesia info | grep "running db nodes"
-```
+    To remove another node from the cluster, call the following command from one of the cluster members:
 
-This command shows all running nodes.
-A healthy cluster should contain all nodes here.
-For example:
-```bash
-running db nodes = [mongooseim@node1, mongooseim@node2]
-```
-To see stopped or misbehaving nodes following command can be useful:
+    ```bash
+    mongooseimctl remove_from_cluster RemoteNodeName
+    ```
 
-```bash
-mongooseimctl mnesia info | grep "stopped db nodes"
-```
+    where `RemoteNodeName` is the name of the node that we'd like to remove from our cluster.
+    This command could be useful when the node is dead and not responding and we'd like to remove it remotely.
+    The successful output from the above command may look like the following:
+
+    ```text
+    The node mongooseim2@localhost has been removed from the cluster
+    ```
+
+### Cluster status
 
-This command shows which nodes are considered stopped.
-This does not necessarily indicate that they are down but might be a symptom of a communication problem.
+=== "CETS"
+
+    Run the command:
+
+    ```bash
+    mongooseimctl cets systemInfo
+    ```
+
+    `joinedNodes` should contain a list of properly joined nodes:
+
+    ```json
+    "joinedNodes" : [
+      "mongooseim@node1",
+      "mongooseim@node2"
+    ]
+    ```
+
+    It should generally be equal to the list of `discoveredNodes`.
+
+    If it is not equal, you could have some configuration or networking issues.
+    You can check the `unavailableNodes`, `remoteNodesWithUnknownTables`,
+    and `remoteNodesWithMissingTables` lists for more information (generally, these lists should be empty).
+    You can read the description for other fields of `systemInfo` in the GraphQL schema file.
```

Comment:
We could just direct to the GraphQL docs, which should have the same description: https://esl.github.io/MongooseDocs/latest/graphql-api/admin-graphql-doc.html#definition-CETSSystemInfo (replace with a relative URL)